zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -1206,20 +1206,20 @@ private void patchParagraph(NotebookSocket conn,
     if (!zConf.isZeppelinNotebookCollaborativeModeEnable()) {
       return;
     }
-    String paragraphId = fromMessage.getType("id", LOG);
+    String paragraphId = fromMessage.getType("id", LOGGER);
```

Comment:
These lines really made me wonder. Passing the logger into a function so that another class can log via it...
However, your change at this point is correct.

Reply:
:)